cosmos/lib/cosmos/models/target_model.rb

```diff
@@ -97,6 +97,11 @@ def self.packets(target_name, type: :TLM, scope:)
       result
     end
 
+    # @return [Array>Hash>] All packet hashes under the target_name
+    def self.all_packet_name_descriptions(target_name, type: :TLM, scope:)
+      self.packets(target_name, type: type, scope: scope).map! { |hash| hash.slice("packet_name", "description") }
```

Comment:
Hash slice just takes the keys you're interested in but still keeps it as a hash